packages/sdk-components-react-remix/src/shared/remix-link.tsx

```diff
@@ -23,9 +23,14 @@ export const wrapLinkComponent = (BaseLink: typeof Link) => {
 
     if (href?.type === "page") {
       let to = href.page.path === "" ? "/" : href.page.path;
+      const urlTo = new URL(to, "https://any-valid.url");
```

Comment:
any chance this could throw? why not simply encodeURIComponent()

Reply:
Because almost no one can cook encodeURI vs encodeURIComponent, with URL we don't need even think about what's is right percent encoding

Reply:
Should not throw because of 2nd parameter